frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java

```diff
@@ -321,6 +321,12 @@ private static Operation getRegisterOperation() {
                         "integer",
                         "100",
                         "Maximum delay for batch aggregation, default: 100."));
+        operation.addParameter(
+                new QueryParameter(
+                        "response_timeout",
+                        "integer",
+                        "2",
+                        "Maximum time, in minutes, the Model Server waits for a response from the model inference code, default: 2."));
```

Comment:
I am wondering if this should be seconds.

Reply:
Agreed, I was thinking the same. I will change this to seconds anyways.